queries/highlights.scm

```diff
@@ -84,41 +125,51 @@
 (octal_number (value) @number)
 (hex_number (value) @number)
 (string) @string
+(boolean) @boolean
 (string_set) @type
 (boolean_set) @type
 (nat_number_set) @type
 (int_number_set) @type
 (real_number_set) @type
 
-; Comments
-(comment) @comment
-(block_comment) @comment
-(single_line) @comment
-
 ; Constants, variables, and operators
 (variable_declaration (identifier) @variable.builtin)
-(constant_declaration (identifier) @constant)
-(constant_declaration (operator_declaration name: (_) @constant))
+(constant_declaration (identifier) @constant.builtin)
+(constant_declaration (operator_declaration name: (_) @constant.bulitin))
 (recursive_declaration (identifier) @operator)
 (recursive_declaration (operator_declaration name: (_) @operator))
 (operator_definition name: (_) @operator)
-(module_definition name: (identifier) @operator)
+(module name: (_) @namespace)
+(extends (identifier_ref) @include)
+(module_definition name: (identifier) @function)
 (function_definition name: (identifier) @function)
 (bound_prefix_op symbol: (_) @function.builtin)
 (bound_nonfix_op (prefix_op_symbol) @operator)
 (bound_infix_op symbol: (_) @function.builtin)
 (bound_nonfix_op (infix_op_symbol) @operator)
 (bound_postfix_op symbol: (_) @function.builtin)
 (bound_nonfix_op (postfix_op_symbol) @operator)
+(pcal_macro name: (identifier) @function.macro)
+(pcal_macro_call name: (identifier) @function.macro)
+(pcal_procedure name: (identifier) @function.macro)
+(pcal_algorithm name: (identifier) @namespace)
+(pcal_process name: (identifier) @function)
+(pcal_var_decl (identifier) @variable.builtin)
+("self") @variable.builtin
```

Comment:
Unit tests picked up that this isn't a valid node type.

Reply:
fixed